services/blocks/standard/handler.go

```diff
@@ -714,6 +714,10 @@ func (s *Service) updateBlobSidecarsForBlock(ctx context.Context,
 	ctx, span := otel.Tracer("wealdtech.chaind.services.blocks.standard").Start(ctx, "updateBlobSidecarsForBlock")
 	defer span.End()
 
+	if !s.blobsSaving {
+		return nil
+	}
+
```

Comment:
Let's avoid creating the span in case that blob saving is disabled.

```suggestion
	if !s.blobsSaving {
		return nil
	}
	
	ctx, span := otel.Tracer("wealdtech.chaind.services.blocks.standard").Start(ctx, "updateBlobSidecarsForBlock")
	defer span.End()
	
```

Reply:
Done!